### DISCUSSION TITLE
#100DaysOfGatsby challenge 1 done?

### DISCUSSION THREAD

**Discussion by hkalic** [gatsbyjs/gatsby#28808] [Community]
Here's my "Challenge 1".
[https://100daysofgatsbymaster07796.gtsb.io/home/](https://100daysofgatsbymaster07796.gtsb.io/home/)

Any suggestions?

Happy New Year every1?

PS: it's not stated in the introduction of this series but you gonna have to know the basics of Github

**Comment by velasco** [comment]
> Here's my "Challenge 1".
> https://100daysofgatsbymaster07796.gtsb.io/home/
> 
> Any suggestions?
> 
> Happy New Year every1?
> 
> PS: it's not stated in the introduction of this series but you gonna have to know the basics of Github



> Here's my "Challenge 1".
> https://100daysofgatsbymaster07796.gtsb.io/home/
> 
> Any suggestions?
> 
> Happy New Year every1?
> 
> PS: it's not stated in the introduction of this series but you gonna have to know the basics of Github

Hi hkalic,

Happy New Year to you, too!.

Looks almost like mine (https://100daysofgatsbymaster31300.gtsb.io/home/). May I ask, how did you handle the location route?

Regards

**Reply by hkalic** [reply]
Well... I just made a directory "location" and inside the city's name *.js. This solution is not really a route but... It's missing the links from one page to other

**Reply by hkalic** [reply]
Here is my code: [https://github.com/hkalic/100days-of-gatsby](https://github.com/hkalic/100days-of-gatsby)

**Reply by velasco** [reply]
Thanks for getting back to me. I adapted your solution. Regards, Francisco

**Reply by giantoctopus** [reply]
I think we may need to look more at https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/ for the routing for the locations. 

I created a location content type in contentful and added three pieces of content via that. Then reviewed the graphql and the above docs to dynamically create those pages using the { } examples in the above doc. I found this useful as well - https://www.youtube.com/watch?v=TaaAPOAuj9w

**Comment by Kevpedia** [comment]
What was the prompt? I had signed up but never got the email

**Reply by hkalic** [reply]
What do you mean by "Prompt"?

**Reply by Kevpedia** [reply]
meaning the first challenge/briefing. I got it, though: https://www.gatsbyjs.com/blog/100days-challenge-1/

**Reply by giantoctopus** [reply]
I received an email titled '[#100DaysOfGatsby] Let's get started!' - gmail pushed it into the promotions tab for me so you may need to check that or possibly spam.
